docs/function/functions.md

```diff
@@ -58,6 +58,7 @@ SELECT UNICODE("AmrDeveloper")
 | CURRENT_DATE      |                  | Date     | Return current date in `YYYY-MM-DD` format.                     |
 | CURRENT_TIMESTAMP |                  | DateTime | Return current date time in `YYYY-MM-DD HH-MM-SS` format.       |
 | MAKEDATE          | Integer, Integer | Date     | Create and return a date based on  a year and a number of days. |
+| NOW               |                  | DateTime | Return current date and time.                                   |
```

Comment:
We need to add it which format the result will be similar to `Return current date time in YYYY-MM-DD HH-MM-SS`format.`